analysis_notebooks/Geography Analysis.ipynb

```diff
@@ -306,154 +311,253 @@
    "metadata": {},
```

Comment:
can't figure out how to make a comment on un-changed code above, but this section is called "Country Counts" -- since that might not always be what is being counted, e.g., regions in gps coordinates, the name should be changed too

Reply:
changed the section name to "Region Counts"